tgui/packages/tgui/interfaces/CommunicationsConsole/BuyingShuttle.tsx

```diff
@@ -76,14 +76,14 @@ function ShuttleCard(props: ShuttleCardProps) {
           }
           tooltip={
             budget < shuttle.creditCost
-              ? `You need ${shuttle.creditCost - budget} more credits.`
+              ? `You need ${shuttle.creditCost - budget} more ${displayed_currency_full_name}.`
               : shuttle.emagOnly
                 ? EMAG_SHUTTLE_NOTICE
                 : undefined
           }
           tooltipPosition="left"
         >
-          {shuttle.emagOnly ? 'Buy' : 'Purchase'}
+          {shuttle.emagOnly ? 'Buy' : `${shuttle.creditCost} ${displayed_currency_name}`}
```

Comment:
Are you able to include the coin icon, like the one next to the right of the `Back` button that is:

`<Icon name="coins" color="gold" />`